app/tasks/ssl.php

```diff
@@ -2,22 +2,35 @@
 
 global $cli;
 
+use Appwrite\Event\Event;
 use Utopia\App;
 use Utopia\CLI\Console;
+use Utopia\Validator\Hostname;
 
 $cli
     ->task('ssl')
     ->desc('Validate server certificates')
-    ->action(function () {
-        $domain = App::getEnv('_APP_DOMAIN', '');
+    ->param('domain', App::getEnv('_APP_DOMAIN', ''), new Hostname(), 'Domain to generate certificate for. If empty, main domain will be used.', true)
+    ->action(function ($domain) {
+        // HTTTP ping to check if domain is Appwrite server
+        $ch = \curl_init();
+        \curl_setopt($ch, CURLOPT_URL, 'http://appwrite/manifest.json');
```

Comment:
For what reason we do this?

Reply:
There was 30s delay previously, to ensure the HTTP server started properly (the first start can take some time).

This pings the Appwrite container HTTP server to ensure it can properly respond. This means it should be also able to properly respond to the HTTP LetsEncrypt challenge in a few seconds, if we schedule a task.

Reply:
Removed check.